### PR TITLE
Add documentation for Condition creates

### PR DESCRIPTION
**Pull request by MadhurRaj**

### CODE REVIEW

File: content/millennium/r4/general-clinical/condition.md [hunk 1]
```diff
@@ -73,6 +73,63 @@ _Implementation Notes_
...
+Location: https://fhir-ehr.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Condition/p12793861
+Last-Modified: Wed, 14 Aug 2019 17:23:14 GMT
+Server-Response-Time: 296.405243
+Status: 201 Created
+Vary: Origin
+X-Request-Id: 11111111111111111111111111111111
+X-Runtime: 2.011826
+</pre>
+
+The `ETag` response header indicates the current `If-Match` version to use on subsequent updates.
```
Comment (nishanthnamana) [inline content/millennium/r4/general-clinical/condition.md]: updates => update.
This was pointed by davis on my PR😋.

File: content/millennium/r4/general-clinical/condition.md [hunk 2]
```diff
@@ -73,6 +73,63 @@ _Implementation Notes_
...
+* Only the body fields mentioned below are supported. Unsupported fields will be ignored.
+* Modifier fields should not be provided, and will cause the transaction to fail.
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, system: true) %>
+
+### Headers
+
+<%= headers head: {Authorization: '&lt;OAuth2 Bearer Token>', Accept: 'application/fhir+json', 'Content-Type': 'application/fhir+json'} %>
```
Comment (nishanthnamana) [inline content/millennium/r4/general-clinical/condition.md]: Do we need Accept header?

File: content/millennium/r4/general-clinical/condition.md (hunk 1 above)
Comment (MadhurRaj) [reply content/millennium/r4/general-clinical/condition.md]: changed: d6f7fbd16cf339fae076b5047a078bd4cdd50682

File: content/millennium/r4/general-clinical/condition.md (hunk 2 above)
Comment (MadhurRaj) [reply content/millennium/r4/general-clinical/condition.md]: removed: d6f7fbd16cf339fae076b5047a078bd4cdd50682

**Pull request merged**